src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -0,0 +1,114 @@
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('about')
+            ->setDescription('Displays information about the current installation')
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        /** @var $kernel KernelInterface */
+        $kernel = $this->getContainer()->get('kernel');
+        $baseDir = realpath($kernel->getRootDir().DIRECTORY_SEPARATOR.'..');
+        $bundles = array_map(function ($bundle) use ($baseDir) {
+            return '* '.$bundle->getName().' (<comment>'.self::formatPath($bundle->getPath(), $baseDir).'</>)';
+        }, $kernel->getBundles());
+        sort($bundles);
+
+        $io->table(array(), array(
+            array('<info>Symfony</>'),
+            new TableSeparator(),
+            array('Version', Kernel::VERSION),
+            array('Version ID', Kernel::VERSION_ID),
+            array('End of maintenance', Kernel::END_OF_MAINTENANCE.(self::isExpired(Kernel::END_OF_MAINTENANCE) ? ' <error>Expired</>' : '')),
+            array('End of life', Kernel::END_OF_LIFE.(self::isExpired(Kernel::END_OF_LIFE) ? ' <error>Expired</>' : '')),
+            new TableSeparator(),
+            array('<info>Kernel</>'),
+            new TableSeparator(),
+            array('Type', get_class($kernel)),
+            array('Name', $kernel->getName()),
+            array('Environment', $kernel->getEnvironment()),
+            array('Debug', $kernel->isDebug() ? 'true' : 'false'),
+            array('Charset', $kernel->getCharset()),
+            new TableSeparator(),
+            array('Root directory', self::formatPath($kernel->getRootDir(), $baseDir)),
+            array('Cache directory', self::formatPath($kernel->getCacheDir(), $baseDir).' (<comment>'.self::formatFileSize($kernel->getCacheDir()).'</>)'),
+            array('Log directory', self::formatPath($kernel->getLogDir(), $baseDir).' (<comment>'.self::formatFileSize($kernel->getLogDir()).'</>)'),
+            new TableSeparator(),
+            array('Bundles (<comment>'.count($bundles).'</>)', implode(PHP_EOL, $bundles)),
+            new TableSeparator(),
+            array('<info>PHP</>'),
+            new TableSeparator(),
+            array('Version', PHP_VERSION),
+            array('Architecture', (PHP_INT_SIZE * 8).' bits'),
+            array('Intl locale', \Locale::getDefault() ?: 'n/a'),
+            array('Timezone', date_default_timezone_get().' (<comment>'.(new \DateTime())->format(\DateTime::W3C).'</>)'),
+            array('OPcache', extension_loaded('Zend OPcache') && ini_get('opcache.enable') ? 'true' : 'false'),
+            array('APCu', extension_loaded('apcu') && ini_get('apc.enabled') ? 'true' : 'false'),
+            array('Xdebug', extension_loaded('xdebug') ? 'true' : 'false'),
```

Comment:
It shows the same info as WDT configuration panel does.. :thinking:  which was the basic idea; to mimic this on CLI, so you can actually spot differences at first sight.

I dropped the bundle list (sounds reasonable), the version ID and an additional table separator.

![image](https://cloud.githubusercontent.com/assets/1047696/24218324/236b6efe-0f43-11e7-9fd5-da7fb2503580.png)

I think it looks pretty clean :))